src/kernel_preprocessor.cpp

```diff
@@ -580,8 +580,8 @@ std::vector<std::string> PreprocessUnrollLoops(const std::vector<std::string>& s
       }
 
       // Retrieves loop information (and checks for assumptions)
-      const auto variable_type = line_split[0];
-      const auto variable_name = line_split[1];
+      const auto& variable_type = line_split[0];  // Can we remove this? The usage is commented but maybe debug?
```

Comment:
I guess you can remove it indeed (and remove the comment). We can always get it back from git history if really necessary.